src/test/java/bio/terra/stairway/QueueFlightTest.java

```diff
@@ -130,4 +132,90 @@ public void admissionControlTest() throws Exception {
     assertThat(
         "1st flight succeeded", flightState.getFlightStatus(), equalTo(FlightStatus.SUCCESS));
   }
+
+  @Test
+  public void supplyQueueTest() throws Exception {
```

Comment:
Is there anything that should be cleaned up after this test?

Reply:
I don't think so. We could clean up the queue, but that it is happily reused just like in the product. We test creation and deletion in the other queue test.